test/system/screenshots_test.rb

```diff
@@ -69,4 +69,13 @@ class ScreenshotsTest < ApplicationSystemTestCase
     visit node.path
     take_screenshot
   end
+  
+  test 'blog page with location modal' do
+    visit '/people'
```

Comment:
@jywarren ...i have changed the URL to people's page . The screenshot will not show the **recent locations** because we are not logged-in .

Reply:
oh ok let me change it back with the log in code too. Let's see!